frontend/src/components/common/NavBar.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useContext } from "react";
+import { HiUser } from "react-icons/hi";
+import { Link } from "react-router-dom";
+
+import logo from "../../ccbc.png";
+import { UserRole } from "../../constants/Enums";
+import AuthContext from "../../contexts/AuthContext";
+
+const NavBar = (): React.ReactElement => {
+  const { authenticatedUser } = useContext(AuthContext);
+  const isAdmin = authenticatedUser?.role === UserRole.Admin;
+
+  return (
+    <div variant="default"> 
```

Comment:
pretty sure this is incorrect syntax ... to use the text style you defined in theme, you need to use the Text component from ChakraUI - check src/components/auth/Login.tsx for ref